README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# duke.Duke project template
```

Comment:
Is this a typo? I think Duke might have changed in some places to duke.Duke after refactoring.